src/Avalonia.Base/Input/PinchEventArgs.cs

```diff
@@ -4,20 +4,32 @@ namespace Avalonia.Input
 {
     public class PinchEventArgs : RoutedEventArgs
     {
-        public PinchEventArgs(double scale, Point scaleOrigin) :  base(Gestures.PinchEvent)
+        public PinchEventArgs(double scale, Point scaleOrigin) : base(Gestures.PinchEvent)
         {
             Scale = scale;
             ScaleOrigin = scaleOrigin;
         }
 
+        public PinchEventArgs(double scale, Point scaleOrigin, double degree) : base(Gestures.PinchEvent)
+        {
+            Scale = scale;
+            ScaleOrigin = scaleOrigin;
+            Angle = degree;
+        }
+
         public double Scale { get; } = 1;
 
         public Point ScaleOrigin { get; }
+
+        /// <summary>
+        /// Pinch angle in degrees
```

Comment:
I think this should be a lot more descriptive. Also needs to start with "Gets" and have a "." at the end to avoid analyzer warnings and follow conventions.

```
/// <summary>
/// Gets the angle of the pinch gesture, in degrees.
/// <summary>
/// <remarks>
/// A pinch gesture is the movement of two pressed points closer together. This property is the measured angle of the line between those two points. Remember zero degrees is a line pointing up.
/// </remarks>
```